File: backend/src/main/java/com/devmatch/backend/domain/project/repository/ProjectRepository.java
package com.devmatch.backend.domain.project.repository;

import com.devmatch.backend.domain.project.entity.Project;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ProjectRepository extends JpaRepository<Project, Long> {

  List<Project> findAllByCreatorId(Long creatorId);
}

File: backend/src/main/java/com/devmatch/backend/domain/project/repository/ProjectRepository.kt
package com.devmatch.backend.domain.project.repository

import com.devmatch.backend.domain.project.entity.Project
import org.springframework.data.jpa.repository.JpaRepository

interface ProjectRepository : JpaRepository<Project, Long> {
    fun findAllByCreatorId(creatorId: Long): List<Project>
}